ui/src/layout/SubMenu.js

```diff
@@ -72,3 +95,8 @@ const SubMenu = ({
 }
 
 export default SubMenu
+
+SubMenu.defaultProps = {
+  secondaryLink: '',
+  dense: false,
```

Comment:
Not related to this pr, but after console logging I found that dense property passed down is always giving undefined and hence always acts as false in the subMenu.

Reply:
These code was originally copied from the React-Admin demo, so it may have some "leftovers". Is this causing any undesirable side-effects?

Reply:
No, no side-effects. Just that getting undefined was unexpected/unpredictable. Removing it doesn't give any change